pkgs/development/python-modules/rerun-sdk/default.nix

```
@@ -12,6 +12,7 @@
   typing-extensions,
   pytestCheckHook,
   python,
+  iconv,
```

```suggestion
  libiconv,
```

I wonder why there is an `iconv` in `all-packages.nix`, but I think it's much more commonly referred to as `libiconv` in Nixpkgs.